engine/src/main/java/com/odysseusinc/arachne/executionengine/service/impl/DescriptorServiceImpl.java

```diff
@@ -147,19 +180,17 @@ private DescriptorBundle findMatchingDescriptor(File dir, Long analysisId, List<
                     return match;
                 });
             }
-        }).map(Map.Entry::getKey).map(this::toBundle).orElse(defaultDescriptorBundle);
+        }).map(Map.Entry::getKey).map(this::toBundle).orElseGet(getDefaultBundle(analysisId, requestedDescriptorId));
```

Comment:
I don't understand why requestedDescriptorId is added here. Default bundle is default bundle. Masquerading it under  requestedDescriptorId is incorrect

Reply:
requestedDescriptorId is used for logging, no masquerading here